templates/vue/packagejsons/assemblyscript/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "near-blank-project",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "build": "npm run build:contract && npm run build:web",
+    "build:contract": "asb",
+    "build:contract:debug": "asb --target debug",
+    "build:web": "node copy-dev-account.js && vue-cli-service build",
+    "dev:deploy:contract": "near dev-deploy",
+    "deploy:contract": "near deploy",
+    "deploy:pages": "gh-pages -d dist/",
+    "deploy": "npm run build && npm run deploy:contract && npm run deploy:pages",
+    "prestart": "npm run build:contract:debug && npm run dev:deploy:contract",
+    "start": "echo The app is starting! It will automatically open in your browser when ready && npm run serve",
+    "dev": "nodemon --watch assembly -e ts --exec \"npm run start\"",
+    "test:web": "vue-cli-service test:unit",
+    "test": "npm run build:contract:debug && asp --nologo && vue-cli-service test:unit",
+    "serve": "node copy-dev-account.js && vue-cli-service serve",
+    "lint": "vue-cli-service lint"
+  },
+  "dependencies": {
+    "near-api-js": "^0.29.0",
+    "core-js": "^3.6.5",
+    "vue": "^2.6.11"
+  },
+  "devDependencies": {
+    "near-cli": "^1.0.1",
+    "near-sdk-as": "^1.0.1",
+    "nodemon": "^2.0.4",
+    "@vue/cli-plugin-babel": "~4.5.0",
+    "@vue/cli-plugin-eslint": "~4.5.0",
+    "@vue/cli-plugin-unit-jest": "^4.5.4",
+    "@vue/cli-service": "~4.5.0",
+    "@vue/test-utils": "^1.0.3",
+    "env-cmd": "^10.1.0",
+    "eslint": "^6.7.2",
+    "eslint-plugin-vue": "^6.2.2",
+    "gh-pages": "^3.0.0",
+    "vue-template-compiler": "^2.6.11"
+  },
+  "eslintConfig": {
```

Comment:
Do we need this same config in the rust package.json?

Reply:
Yes, you're right. I completely re-did `eslintConfig` and put it in both packages. I also added the needed eslint config to check indentation in vue templates